fastled_delay.h

```diff
@@ -34,6 +34,12 @@ template<int WAIT> class CMinWait {
 
 // Default is now just 'nop', with special case for AVR
 
+// ESP32 core has it's own definition of NOP, so undef it first
+#ifdef ESP32
+#undef NOP
```

Comment:
Gyah -- there's a reason why, with the CRGBW/CRGB16 re-write I'm re-doing all the macros to be prefaced with FASTLED_ or FL_ ...